core/io/file_access.cpp

```diff
@@ -57,13 +57,14 @@ Ref<FileAccess> FileAccess::create(AccessType p_access) {
 }
 
 bool FileAccess::exists(const String &p_name) {
-	if (PackedData::get_singleton() && !PackedData::get_singleton()->is_disabled() && PackedData::get_singleton()->has_path(p_name)) {
+	const String name = ResourceUID::ensure_path(p_name);
```

Comment:
I haven't looked at the details so I might be wrong, but it seems incorrect to me that FileAccess would need to know about ResourceUIDs. FileAccess is the low level file API that deals with the actual files on the system. Abstractions like ResourceUIDs should be used via ResourceLoader instead.

Reply:
#99286 exists though.

Reply:
I stand corrected then :D